packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/portabilities.service.js

```diff
@@ -1,18 +1,87 @@
-import find from 'lodash/find';
-import forEach from 'lodash/forEach';
+export default class TelecomTelephonyAliasPortabilitiesService {
+  /* @ngInject */
+  constructor($http, $q) {
+    this.$http = $http;
+    this.$q = $q;
+  }
 
-export default function groupPortaByNumbers(portabilities) {
-  const numbers = [];
-  forEach(portabilities, (portability) => {
-    forEach(portability.numbersList, (number) => {
-      numbers.push({
-        number,
-        portability,
-        lastStepDone: find(portability.steps.slice().reverse(), {
-          status: 'done',
-        }),
+  fetchPortability(billingAccount) {
+    return this.$http
+      .get(`/telephony/${billingAccount}/portability`)
+      .then(({ data: ids }) => {
+        return this.$q.all(
+          ids.map((id) => {
+            return this.$http
+              .get(`/telephony/${billingAccount}/portability/${id}`)
```

Comment:
Not possible to use Iceberg expand instead ? To avoid N+1 call

Reply:
It's applied.